clients/v1_auth/v1_auth.go

```diff
@@ -0,0 +1,428 @@
+	if err1 != nil || err2 != nil {
+		return fmt.Errorf("could not find Authorization from ID %q: %w", id, err1)
+	}
+
+	usr, err := c.UsersApi.GetUsersID(ctx, auth.GetUserID()).Execute()
+	if err != nil {
+		return fmt.Errorf("could not find user from user ID %q: %w", auth.GetUserID(), err)
+	}
+
+	ps := make([]string, 0, len(auth.GetPermissions()))
+	for _, p := range auth.GetPermissions() {
+		ps = append(ps, permString(p))
+	}
+
+	return c.printV1Tokens(&v1PrintOpts{
+		deleted: true,
+		token: &v1Token{
+			ID:          auth.GetId(),
+			Description: auth.GetDescription(),
+			Token:       auth.GetToken(),
+			Status:      auth.GetStatus(),
+			UserName:    usr.GetName(),
+			UserID:      usr.GetId(),
+			Permissions: ps,
+		},
+	})
+}
+
+type ListParams struct {
+	clients.OrgParams
+	clients.AuthLookupParams
+	User   string
+	UserID string
+}
+
+func (c Client) List(ctx context.Context, params *ListParams) error {
+	req := c.AuthorizationsApi.GetAuthorizations(ctx)
+
+	if params.User != "" {
+		req = req.User(params.User)
+	}
+	if params.OrgID.Valid() {
+		req = req.OrgID(params.OrgID.String())
+	}
+	if params.OrgName != "" {
+		req = req.Org(params.OrgName)
+	}
+	var id string
+	if params.AuthLookupParams.IsSet() {
+		newId, err := c.getAuthReqID(ctx, &params.AuthLookupParams)
+		if err != nil {
+			return fmt.Errorf("could not get User from Username %q: %w", params.Username, err)
+		}
+		id = newId
+	}
+	if params.UserID != "" {
+		id = params.UserID
+	}
+	if id != "" {
+		req = req.UserID(id)
+	}
+
+	auths, err := req.Execute()
+	if err != nil {
+		return fmt.Errorf("could not find Authorizations for specified arguments %w", err)
+	}
+
+	var tokens []v1Token
+	for _, a := range auths.GetAuthorizations() {
+		var permissions []string
+		for _, p := range a.GetPermissions() {
+			permissions = append(permissions, permString(p))
+		}
+
+		usr, err := c.UsersApi.GetUsersID(ctx, a.GetUserID()).Execute()
+		if err != nil {
+			return fmt.Errorf("could not find user with ID %q: %w", a.GetUserID(), err)
+		}
+
+		tokens = append(tokens, v1Token{
+			ID:          a.GetId(),
+			Description: a.GetDescription(),
+			Token:       a.GetToken(),
+			Status:      a.GetStatus(),
+			UserName:    usr.GetName(),
+			UserID:      usr.GetId(),
+			Permissions: permissions,
+		})
+	}
+
+	return c.printV1Tokens(&v1PrintOpts{tokens: tokens})
+}
+
+type ActiveParams struct {
+	clients.AuthLookupParams
+}
+
+func (c Client) SetActive(ctx context.Context, params *ActiveParams, active bool) error {
+	id, err := c.getAuthReqID(ctx, &params.AuthLookupParams)
+	if err != nil {
+		return err
+	}
+
+	req := c.AuthorizationsApi.PatchAuthorizationsID(ctx, id)
+	var status string
+	if active {
+		status = "active"
+	} else {
+		status = "inactive"
+	}
+	req.AuthorizationUpdateRequest(api.AuthorizationUpdateRequest{
+		Status: &status,
+	})
+
+	auth, err := req.Execute()
+	if err != nil {
+		return fmt.Errorf("could not find Authorization with ID %q: %w", id, err)
```

Comment:
Since this is a PATCH, not a GET:
```suggestion
		return fmt.Errorf("failed to update status of authorization with ID %q: %w", id, err)
```